crates/chain-config/src/config/coin.rs

```diff
@@ -157,6 +121,54 @@ impl GenesisCommitment for Coin {
     }
 }
 
+#[cfg(feature = "test-helpers")]
+pub mod coin_config_helpers {
```

Comment:
This code not only used in tests, it is used in production in the case of state config.

Reply:
I see it used in 2 places in `chain-config/src/config/state.rs`, `random_testnet()` and `local_testnet()` both of which are hidden behind test-helpers. is the code you're talking about in a different repo?

Reply:
Hmm, maybe we don't use it anymore in the state config, then okay=) It would be nice to put the section in decompression under the same feature flag too then

Reply:
cool

Reply:
addressed in [e9eb633](https://github.com/FuelLabs/fuel-core/pull/2593/commits/e9eb633a3323fa292f310f0233f3d9d90cae77f3)